licensing/src/main/java/io/crate/license/DecryptedLicenseData.java

```diff
@@ -31,18 +32,28 @@
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 import java.util.Objects;
+import java.util.Optional;
 
 public class DecryptedLicenseData {
 
     private static final String EXPIRATION_DATE_IN_MS = "expirationDateInMs";
     private static final String ISSUED_TO = "issuedTo";
+    private static final String SIGNATURE = "signature";
 
     private final long expirationDateInMs;
     private final String issuedTo;
+    private Optional<String> signature;
```

Comment:
We're usually going with `Nullable` annotation and nullable objects instead of using `Optional`